base_layer/wallet/src/schema.rs

```diff
@@ -24,6 +24,7 @@ table! {
         message -> Text,
         timestamp -> Timestamp,
         acknowledged -> Integer,
+        isread -> Integer,
```

Comment:
`is_read` 

Shouldn't this be a Bool?

Reply:
There is no default boolean type in SQLLite:

https://www.sqlite.org/datatype3.html